UI/MVUX/MVUX/MVUX/Presentation/SelectionSample/SelectionPage.xaml

```diff
@@ -0,0 +1,44 @@
+﻿<Page x:Class="MVUX.Presentation.SelectionSample.SelectionPage"
```

Comment:
This page show how selection works but the FeedPage also says it demonstrated Selection. How come we do selection in the FeedPage as well?

Reply:
This one should be the one showcasing Selection, the other one should mention nothing about Selection.